src/controllers/ClamAVController.js

```diff
@@ -59,11 +108,10 @@ function scan (req, res, next) {
 function check (req, res) {
   clamav.ping(config.CLAMAV_PORT, config.CLAMAV_HOST, 2000, err => {
     if (err) {
-     console.log( err )

```

Comment:
Let's keep the `console.log` here - in case the check fails, would be nice to know why

Reply:
I will change it to `logger.error(err)`